.storybook/ResetCss.js

```diff
@@ -0,0 +1,63 @@
+import { createGlobalStyle } from 'styled-components'
+
+export const ResetCss = createGlobalStyle`
+  html, body, div, span, applet, object, iframe,
+  h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+  a, abbr, acronym, address, big, cite, code,
+  del, dfn, em, img, ins, kbd, q, s, samp,
+  small, strike, strong, sub, sup, tt, var,
```

Comment:
Added the reset file as a separate one as the plan is to move it inside S'mores at some point